app/views/users/show.html.erb

```diff
@@ -1,2 +1,48 @@
-<h1>User Details</h1>
-<h3>User ID: <%= @user.id %></h3>
+<div class="container mt-5">
+  <div class="row">
+    <div class="col-3">
+      <div class="card border-2 border-dark">
+        <img src="<%= @user.photo %>" alt="user" style="height: 100px;" />
+      </div>
+    </div>
+
+    <div class="col-9">
+      <div class="card border-2 border-dark">
+      <div class="card-body">
+      <div class="card-title">
+        <h3 class="m-0"><b><%= @user.name %></b></h3>
+      </div>
+      <div class="d-flex h-50 justify-content-end align-items-end">
+        <p class="m-0">Number of posts: <%= @user.posts_counter %></p>
+      </div>
+    </div>
+      </div>
+    </div>
+  </div>
+</div>
```

Comment:
- [ ] I noticed that this piece of code is similar to the one we have in the `_user_card.html.erb`. I think it would be better to follow the DRY principle and avoid repeating the same piece of code.  Utilizing a partial in this context would not only eliminate code duplication but also enhance the maintainability and consistency of the application. and consistency throughout the application 💪 What do you think about it? ✨

Reply:
well resolved! ✔️

Reply:
Thank you.